backend/lib/webserver/ValetudoRouter.js

```diff
@@ -139,6 +141,29 @@ class ValetudoRouter {
         });
     }
 
+    initSSE() {
```

Comment:
This will also need a shutdown function similar to what can be seen here:

https://github.com/Hypfer/Valetudo/blob/2d38a929c997e82e733708ccf90896d3e3e6105b/backend/lib/webserver/RobotRouter.js#L139-L143

https://github.com/Hypfer/Valetudo/blob/2d38a929c997e82e733708ccf90896d3e3e6105b/backend/lib/webserver/WebServer.js#L213


as open SSE connections will prevent a successful Valetudo shutdown.
That's the cause of the forced shutdowns we were seeing often when replacing binaries.